lib/views/status-indicator.js

```diff
@@ -43,4 +46,14 @@ export default class StatusIndicator {
     this.properties = properties
     return etch.update(this)
   }
+
+  readAfterUpdate () {
```

Comment:
I don't immediately understand how this name refers to what is happening inside of it. Rename it? Let's embrace verbose function names 🤘